src/Block/Types.hs

```diff
@@ -11,17 +11,19 @@ module Block.Types (
     AlonzoBlock (..),
     AlonzoTransaction (..),
     OgmiosResponse (..),
+    TxOut (..),
 ) where
 
-import Data.Aeson (FromJSON, ToJSON, withObject, (.:))
+import Data.Aeson (FromJSON, ToJSON, withObject, (.:), (.:?))
 import Data.Aeson qualified as Json
 import Data.HashMap.Strict qualified as HM
```

Comment:
Is this import redundant now?

Reply:
Just in case it's not clear: I mean the `HashMap` one

Reply:
It's not: https://github.com/mlabs-haskell/ogmios-datum-cache/pull/24/files/64aa70bd891f5e68dd5385aa6157408b91c4be36#diff-db642f055f2fdbe5056ad04262def6c773995c8896b99bf68466ef1dcc6396aaR212